website/src/_includes/layouts/documentation.njk

```diff
@@ -26,7 +26,7 @@ layout: layouts/base.njk
         <li><a href="/plugins/prettier">Prettier</a></li>
         <li><a href="/plugins/biome">Biome (JS/TS/JSON)</a></li>
         <li><a href="/plugins/ruff">Ruff (Python)</a></li>
-        <li><a href="/plugins/jupyter">Juypter</a></li>
+        <li><a href="/plugins/jupyter">Jupyter</a></li>
```

Comment:
Looks like this is the last one: https://github.com/search?q=org%3Adprint%20juypter&type=code

Thanks!